src/main/java/io/confluent/connect/hdfs/HdfsSinkTask.java

```diff
@@ -69,6 +69,8 @@ public void start(Map<String, String> props) {
     } catch (ConfigException e) {
       throw new ConnectException("Couldn't start HdfsSinkConnector due to configuration error.", e);
     } catch (ConnectException e) {
+      log.info("Couldn't start HdfsSinkConnector due to {}",  e.getMessage());
```

Comment:
Let's just use the `log.info("Couldn't start HdfsSinkConnector:",  e);` since that will give the stack trace as well.